TerriaJS/terriajs
# Make clickable moment points optional.

### What this PR does

Partial fix for https://github.com/TerriaJS/de-australia-map/issues/94

Right now the moment points expects an `onClick` handler and throws a console error if not set. This change makes it optional. 

### Testing
1. Visit this [next branch link](http://ci.terria.io/next/#share=s-100uNrxDtubTYVMIWovjoukHywV)
2. Observe, when hovering the dot points, they appear clickable and throws a console error when clicking.
3. Now, visit [this link](http://ci.terria.io/non-clickable-moment-points/#share=s-oab4cshjiUvcvgfzIUEvsipmJXP).
4. Observe, when hovering the dot points, they are not clickable and also the fill color changes on hover.

### Checklist
